play/main.ts

```diff
@@ -1,8 +1,11 @@
 import { createApp } from 'vue'
+import ElementPlus from 'element-plus'
```

Comment:
We can auto import this without import manually

Reply:
Yes, The modification here was accidentally caused, because `unplugin-vue-components` does not work in my environment, there should be some problem somewhere else